env-example.toml

```diff
@@ -24,6 +24,9 @@ access_token = "docker hub access token"
 [runners."cluster:k8s"]
 pod_resource_cpu      = "100m"
 pod_resource_memory   = "100Mi"
+sysctls = [
```

Comment:
Merged from https://github.com/testground/testground/pull/1008 which @nonsense made for this purpose.

This change makes the default sysctls empty unless other specified in env.toml.